packages/semantic-release/config/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@buildit/semantic-release",
+  "version": "1.0.0",
+  "description": "> TODO: description",
+  "author": "Ash Blue <ash@blueashes.com>",
+  "homepage": "https://github.com/buildit/gravity-ui-sass#readme",
```

Comment:
Minor: this and the description should probably be updated ;)

Reply:
...and the homepage. Ought to be a link to this package's README as opposed to the monorepo's root README.